.travis.yml

```diff
@@ -50,13 +49,17 @@ jobs:
           fi
         fi
         npm run ci:sauce
+    if: type = pull_request OR branch = master
   - stage: deploy
-    script:
-    - sudo apt-get update
-    - sudo apt-get install default-jre rsync
-    - wget https://artifacts.crowdin.com/repo/deb/crowdin.deb -O crowdin.deb
-    - sudo dpkg -i crowdin.deb
-    - sleep 5
+    addons:
```

Comment:
Speed up installation

Reply:
using this way in past, but failed get GPG key. And fixed on #732 .